### ISSUE TITLE
EventMachine dependency?

### ISSUE THREAD

**Issue by brendonrapp** [joshwand/campfire-bot#5]
Does the EventMachine dependency serve a purpose? There don't appear to be any calls to EventMachine::run and the only place I even see a reference to EventMachine is the require line.

Fired up a quick test bot and it seems to work even with the EventMachine require line commented out.

Am I missing something?

**Comment by joshwand** [comment]
yeah I can probably remove this... since it's actually a tinder dependency.
